helpers/hcache_simple.py

```diff
@@ -461,6 +461,7 @@ def reset_mem_cache(func_name: str = "") -> None:
 
 
 def reset_disk_cache(func_name: str = "") -> None:
+    # Question(Sai) : Why are we doing "assert 0" ? 
```

Comment:
We don't want to blast away the cache by mistake.

You can replace the assert 0 with hsystem.query_yes_no(msg)